static/css/main.css

```diff
@@ -223,424 +225,422 @@ body {
   width: 360px;
 }
 
-.loading .pill.rawbg {
-  display: none;
+.loading .bgButton .rawnoise {
+    display: none;
 }
 
-.pill.rawbg {
-  display: inline-block;
-  border-radius: 2px;
-  border: 1px solid #808080;
-  padding: 0;
+.bgButton .rawnoise {
+    display: inline-block;
+    border-radius: 2px;
+    border: 1px solid #808080;
 }
 
-.pill.rawbg em {
-  color: white;
-  background-color: black;
-  display: block;
-  font-size: 20px;
-  border-radius: 0;
-  padding: 0 2px;
+.bgButton .rawnoise em {
+    color: white;
+    background-color: black;
+    display: block;
+    font-size: 20px;
 }
 
-.pill.rawbg label {
-  display: block;
-  font-size: 14px;
-  padding: 0 2px;
+.bgButton .rawnoise label {
+    display: block;
+    font-size: 14px;
 }
 
 .alarming .bgButton {
-  border-color: #bdbdbd;
-  color: #000;
-  box-shadow: 2px 2px 0 #ddd;
+    border-color: #bdbdbd;
+    color: #000;
+    box-shadow: 2px 2px 0 #ddd;
 }
 
-.alarming.urgent .bgButton {
-  background-color: red;
+.alarming .bgButton.urgent {
+    background-color: red;
 }
 
-.alarming.warning .bgButton {
-  background-color: yellow;
+.alarming .bgButton.warning {
+    background-color: yellow;
 }
 
 .alarming .bgButton:active {
-  border: 2px solid #bdbdbd;
-  box-shadow: none;
-  -moz-box-shadow: none;
-  -webkit-box-shadow: none;
+    border: 2px solid #bdbdbd;
+    box-shadow: none;
+    -moz-box-shadow: none;
+    -webkit-box-shadow: none;
 }
 
 
 .button {
-  text-align: center;
-  background: #ababab;
-  margin: 10px auto;
-}
-
-.tooltip {
-  top: 0;
-  left: 0;
-  color: #444;
-  position: absolute;
-  text-align: left;
-  padding: 4px;
-  font-size: 14px;
-  line-height: 15px;
-  background: white;
-  border: 0;
-  border-radius: 8px;
-  float: right;
+    text-align: center;
+    background: #ababab;
+    margin: 10px auto;
+}
+
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Element (div.tooltip) is overqualified, just use .tooltip without element name.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/969293254/issues/source?bid=2452374&fileBranchId=2861014#l278)